app/views/media_library/index.html.erb

```diff
@@ -22,13 +22,23 @@
     <div class="small-12 small-centered columns -fix">
       <div id="masonry-layout" class="masonry-layout" hidden data-columns>
         <% @media_contents.each do |media| %>
+          <%
+            puts case media.type
+              when MediaContent::TYPE_ALBUM
+                picture_url = media.photos.first.photo_sizes.where(size: PhotoSize::MEDIUM).first.url
+              when MediaContent::TYPE_PHOTO
+                picture_url = media.photo_sizes.where(size: PhotoSize::MEDIUM).first.url
+              else
+                picture_url = nil
+              end
+          %>
```

Comment:
@simaob I put this code in the MediaContent model, but I got a rare error when I use in this loop. I'll like to see tomorrow with you.